crates/bcr-wdc-ebpp/src/service.rs

```diff
@@ -191,3 +227,10 @@ where
         Ok(response)
     }
 }
+
+fn validate_ebill_request_signature(
+    signed: &SignedRequestToMintFromEBillDesc,
+) -> Result<&RequestToMintFromEBillDesc> {
+    // TODO: Implement the signature validation logic
```

Comment:
Once the e-bill validation logic is settled in bcr-ebill-core, you could maybe re-use some of it here :+1: